### PR TITLE
Enable datadog 

### PR DESCRIPTION
**Pull request by gpontejos**
Oops. Not sure why I did that. I think this should fix it.

### CODE REVIEW

File: broker-deploy/app-instance-launch.yml [hunk 1]
```diff
@@ -152,16 +152,18 @@
...
     # While Broker's Git branch for development work is called 'development',
     # the BRANCH variable tends to be used to represent an "environment" in this playbook, so often is 'dev'
-    - name: Disable datadog in Dev
+    - name: Disable datadog apm if env is sandbox or dev
+      become: true
+      lineinfile:
+        dest: "/etc/datadog-agent/datadog.yaml"
+        regexp: "^  enabled: true"
+        insertafter: "^apm_config:.*"
+        line: "  enabled: false"
```
Comment (bminahan-kc) [inline broker-deploy/app-instance-launch.yml:161]: False

File: broker-deploy/app-instance-launch.yml [hunk 2]
```diff
@@ -173,7 +175,7 @@
             dest: "/data-act/backend/dataactvalidator/app.py"
             regexp: "^tracer\\.enabled = False"
             replace: "tracer.enabled = True"
-      when: BRANCH == "dev"
+      when: BRANCH != "dev"
```
Comment (bminahan-kc) [inline broker-deploy/app-instance-launch.yml:178]: so if BRANCH == sandbox it will be set to False then True

Comment (gpontejos) [reply broker-deploy/app-instance-launch.yml]: Updated -- added sandbox.

File: broker-deploy/app-instance-launch.yml (hunk 1 above)
Comment (bminahan-kc) [reply broker-deploy/app-instance-launch.yml:161]: nevermind I was wrong here

**Pull request merged**